scripts/docker/gatkbase/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.c
     apt-get -y clean  && \
     apt-get -y autoclean  && \
     apt-get -y autoremove && \
+    rm -rf /usr/lib/google-cloud-sdk/bin/anthoscli /usr/lib/google-cloud-sdk/platform/anthoscli_licenses /usr/lib/google-cloud-sdk/platform/bundledpythonunix && \
```

Comment:
Add a comment explaining why these are explicitly removed

Reply:
Done.  They're giant, we don't use them, and removing them doesn't seem to cause problems.